hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -970,6 +970,27 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 	return ctrl.Result{}, nil
 }
 
+func isArchAndPlatformSupported(nodePool *hyperv1.NodePool) bool {
+	supported := false
+
+	switch nodePool.Spec.Platform.Type {
+	case hyperv1.AWSPlatform:
+		if nodePool.Spec.Arch == "amd64" || nodePool.Spec.Arch == "arm64" {
```

Comment:
Can you replace the text versions of the CPU arches with the constants in nodepool_types.go please? - https://github.com/openshift/hypershift/blob/cffe1be765e592bbff09c77b51b59b43c0a85c4b/vendor/github.com/openshift/hypershift/api/hypershift/v1beta1/nodepool_types.go#L16